src/frontend/components/App.jsx

```diff
@@ -3,16 +3,12 @@ import {render} from 'react-dom';
 import NewMemberForm from './NewMemberForm.jsx';
 import { Resources } from '../config/strings';
 
-export default class App extends Component {
-    render() {
-        return (
-          <div className="container">
-            <div id="form" className="form-container">
-                <NewMemberForm />
-            </div>
-          </div>
-        )
-    }
-}
+const App = () => (
+  <div className="container">
+    <div id="form" className="form-container">
+      <NewMemberForm />
+    </div>
+  </div>
+);
```

Comment:
Functional components FTW. Also, I couldn't see this being used anywhere outside this file, so I un-exported it.